src/components/donations/grid/GridAppbar.tsx

```diff
@@ -16,6 +20,14 @@ const addIconStyles = {
 export default function GridAppbar() {
   const router = useRouter()
   const { t } = useTranslation()
+  const exportToExcel = useMutation({
+    mutationFn: useExportToExcel(),
+    onError: () => AlertStore.show(t('common:alerts.error'), 'error'),
+    onSuccess: ({ data }) => {
+      downloadFile('Donations.xls', data)
```

Comment:
the extension should be xlsx, because the file contains xlsx and gives error when opened as xls

Reply:
@igoychev Changed directly on master branch

Thanks for pointing out